app/mailers/objectives_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ObjectivesMailer < ApplicationMailer
+  def objectives_achieved(objective, account_share = nil)
+    @name = objective.account.name
+    @user = objective.account.parent.user
+    @balance = objective.account.balance
+    @objective = objective.name
+    prevent_delivery and return unless @user
+
+    mail to: account_share&.email || @user.email, title: "Goal #{@objective} achived"
```

Comment:
See here how to send email to multiple recipients - https://stackoverflow.com/a/7437303